test/functional/page_objects/discover_page.ts

```diff
@@ -483,31 +481,26 @@ export class DiscoverPageObject extends FtrService {
     });
   }
 
-  public async clickCreateNewDataView() {
-    await this.retry.waitForWithTimeout('data create new to be visible', 15000, async () => {
-      return await this.testSubjects.isDisplayed('dataview-create-new');
+  public async createDataView(dataViewName: string) {
+    await this.clickIndexPatternActions();
+    await this.unifiedSearch.clickCreateNewDataView();
+    await this.testSubjects.setValue('createIndexPatternTitleInput', dataViewName, {
+      clearWithKeyboard: true,
+      typeCharByChar: true,
     });
-    await this.testSubjects.click('dataview-create-new');
-    await this.retry.waitForWithTimeout(
-      'index pattern editor form to be visible',
-      15000,
-      async () => {
-        return await (await this.find.byClassName('indexPatternEditor__form')).isDisplayed();
-      }
-    );
-    await (await this.find.byClassName('indexPatternEditor__form')).click();
+    await this.testSubjects.click('saveIndexPatternButton');
```

Comment:
We better use the following to prevent flakyness:

```
    await this.unifiedSearch.createNewDataView(name, false, hasTimeField);
    await this.retry.waitFor('flyout to get closed', async () => {
      return !(await this.testSubjects.exists('indexPatternEditor__form'));
    });
```